src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheCrusader/boss_northrend_beasts.cpp

```diff
@@ -321,26 +321,20 @@ class npc_snobold_vassal : public CreatureScript
 
         struct npc_snobold_vassalAI : public ScriptedAI
         {
-            npc_snobold_vassalAI(Creature* creature) : ScriptedAI(creature)
+            npc_snobold_vassalAI(Creature* creature) : ScriptedAI(creature), _instance(creature->GetInstanceScript()), _isActive(false)
             {
-                _instance = creature->GetInstanceScript();
                 _instance->SetData(DATA_SNOBOLD_COUNT, INCREASE);
             }
 
-            void Initialize()
```

Comment:
Initialize is used to avoid warnings from uninitialized variables as the checker cant tell that Reset is called after create... so yeah use Initialize to initialze variables as boolean or integers xD